### PR TITLE
[CRE] Added rage columns for WWRift

### PR DESCRIPTION
**Pull request by bavovanachte**

### CODE REVIEW

**Comment by bavovanachte** [comment]
For the WWRift competition I added these along with competition score on my fork. The competition score isn't useful outside of the competition, but this might be.

File: src/main/cre.js [hunk 1]
```diff
@@ -184,14 +184,24 @@ function showPop(type) {
   function getHeaderRow() {
     var headerHTML;
     if (locationName === "Event") {
-      headerHTML =
-        "<tr align='left'><th align='left'>Mouse Name</th><th data-filter='false'>Catch Rate</th><th data-filter='false'>Gold</th><th data-filter='false'>Points</th><th data-filter='false'>Min Luck</th>";
```
Comment (bavovanachte) [inline src/main/cre.js]: This part is not related to any function change, just OCD'ing on these 2 >120 character lines. If you prefer I can remove them here and issue a separate PR for them.

**Comment by tsitu** [comment]
Thanks for the PR! Currently traveling, so I will try to merge it in when I get some downtime to check it out.

**Comment by bavovanachte** [comment]
I'm not in any sort of hurry with this. Whenever you have the time is fine.

**Pull request merged**